filenames.gypi

```diff
@@ -17,6 +17,7 @@
       'lib/browser/api/content-tracing.js',
       'lib/browser/api/dialog.js',
       'lib/browser/api/exports/electron.js',
+      'lib/browser/api/module-list.js',
```

Comment:
Looks like this list is sorted so should this go after ` 'lib/browser/api/menu-item-roles.js',` instead?